.changeset/stale-teachers-live.md

```diff
@@ -0,0 +1,6 @@
+---
+'@keystonejs/app-admin-ui': patch
+'@arch-ui/modal-utils': patch
+---
+
+Upgrade react-transition-group to 4.3.0
```

Comment:
```suggestion
Upgraded react-transition-group to 4.3.0.
```

Reply:
It's a small thing but we're trying to consistently use past-tense and full sentences in changesets.

Reply:
Noted 👍